### PR TITLE
CVE-2007-5000

### PR DESCRIPTION
**Pull request by aldenreed**

### CODE REVIEW

File: cves/CVE-2007-5000.yml [hunk 1]
```diff
@@ -3,15 +3,15 @@ CWE_instructions: |
...
   If you are manually editing this file, then you are "curating" it. Set the
   entry below to "true" as soon as you start. This will enable additional
   integrity checks on this file to make sure you fill everything out properly.
   If you are a student, we cannot accept your work as finished unless curated is
   set to true.
-curated: false
+curated: true
 announced_instructions: |
   Was there a date that this vulnerability was announced to the world? You can
   find this in changelogs, blogs, bug reports, or perhaps the CVE date. A good
```
Comment (Pearse4004) [inline cves/CVE-2007-5000.yml:17]: Possibly the date format could be an issue

File: cves/CVE-2007-5000.yml [hunk 2]
```diff
@@ -32,7 +32,10 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: |
+  The mod_imagemap part of this program had a vulnerability where an attacker
+  could inject malicious HTML/Javascript code into the input in order to compromise the 
+  integrity of the system.
```
Comment (Pearse4004) [inline cves/CVE-2007-5000.yml:38]: Good explanation very clear

File: cves/CVE-2007-5000.yml [hunk 3]
```diff
@@ -42,27 +45,17 @@ bounty:
   announced:
   url:
 reviews: []
```
Comment (Pearse4004) [inline cves/CVE-2007-5000.yml:47]: Was there really no reviews for it?

File: cves/CVE-2007-5000.yml [hunk 4]
```diff
@@ -140,12 +135,10 @@ major_events:
...
     we want to capture what the development team was dealing with at the time.
-  answer:
+  answer: There were some files that had a name change during development.
   events:
-    - name:
-      date:
-    - name:
-      date:
+    - name: mod_imap renamed to mod_imagemap
+      date: April 6, 2005
```
Comment (Pearse4004) [inline cves/CVE-2007-5000.yml:141]: Format of date might have to be yyyy-mm-dd

File: cves/CVE-2007-5000.yml (hunk 1 above)
Comment (aldenreed) [reply cves/CVE-2007-5000.yml:17]: I'm assuming you're talking about line 21, in which case I'll change that. I don't know how that happened though.

File: cves/CVE-2007-5000.yml (hunk 4 above)
Comment (aldenreed) [reply cves/CVE-2007-5000.yml:141]: Thank you. I'll change it.

File: cves/CVE-2007-5000.yml (hunk 3 above)
Comment (aldenreed) [reply cves/CVE-2007-5000.yml:47]: There didn't seem to be any.

**Pull request closed without merging**